scenarioo-client/app/components/detailarea/detailarea.component.html

```diff
@@ -15,41 +15,83 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<div class="sc-meta-data-panel row">
+<div class="details-area-container">
 
-    <button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic"
-            class="btn btn-primary pull-right" id="sc-showHideDetailsButton" type="button">
-        {{ isPanelCollapsed ? "Show details" : "Hide details" }}
-    </button>
-
-    <div [ngClass]="isPanelCollapsed ? 'col-sm-11' : 'col-sm-8'">
-        <ng-content></ng-content>
+    <!-- Top Level Button Row -->
+    <div class="row">
+        <button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic"
+                class="btn btn-primary pull-right" id="sc-showHideDetailsButton" type="button">
+            {{ isPanelCollapsed ? "Show details" : "Hide details" }}
+        </button>
     </div>
 
-    <div #metaDataPanel class="detail-container pull-right"  [ngClass]="isPanelCollapsed ? '' : 'col-sm-4'">
-        <div class="detail-row">
-            <div (click)="togglePannelCollapsedValue()" class="detail-cell detail-button">
-                <span *ngIf="!isPanelCollapsed">
-                    <i class="fas arrowCenter fa-angle-right"></i>
-                </span>
-                <span *ngIf="isPanelCollapsed">
-                    <i class="fas arrowCenter fa-angle-left"></i>
-                </span>
-            </div>
+    <!-- Content Area -->
+    <div class="row flex">
+
+
+        <!-- Page Content left (will fill if details are collapsed) -->
+        <div class="detailarea-page-content"
+             [ngClass]="isPanelCollapsed ? 'col-sm-11 flex-fill' : 'col-sm-8'">
```

Comment:
The full width could be taken up by setting _col-sm-12_ as well (instead of _col-sm-11 flex-fill_)

Reply:
Yes, I tried that - but then it looked ugly - it's why this seemed like a good compromise for now - at least they seem to work well together. I see no troubles currently. I have even seen a stackoverflow post were they proposed some similar mixes in bootstrap 4 ("col-sm-11" with "flex-grow-1" - unfortunately we do not have bootstrap 4 yet, but at least this is already almost similar. Maybe we should go for bootstrap 4 ??